### ISSUE TITLE
Add supercontigs to feature contigs that transfer-sequences processes

### ISSUE THREAD

**Issue by sammyjava** [legumeinfo/mine-issues#134]
Update core transfer-sequences to include features that live on supercontigs. We'll not have chromosome-only redlining in our genomic communities!

**Comment by sammyjava** [comment]
Yikes, Chromosome is hardcoded ALL OVER this post-processor. I think I may wait on this, it's a pretty big update.

**Comment by sammyjava** [comment]
Actually wasn't too bad, since Kim Rutherford wrote it. I'll close this since it's been merged into 5.1.0.3 and 5.1.0.4. I'll look into parallel processing, but I think most the time is spent on queries and stores.
